lib/http.js

```diff
@@ -31,11 +31,22 @@ exports.request = function(options, cb) {
   return new ClientRequest(options, cb);
 };
 
-exports.get = function(options, cb) {
-  var req = exports.request(options, cb);
-  req.end();
-  return req;
-};
+function shortcutMethod(method) {
+  return function(options, cb) {
+    options['method'] = method;
+    var req = exports.request(options, cb);
```

Comment:
Yeah, in my current code I have
`'Content-Length': body ? Buffer.byteLength(body) : 0,`

Then later on:
`body && req.write(body);`

But for now just want to get some sort of idea if people even want this new shortcut feature implemented.